pypeit/scripts/coadd_2dspec.py

```diff
@@ -159,6 +159,8 @@ def main(args):
         parset['calibrations']['wavelengths']['refframe'] = 'observed'
         # TODO Flexure correction for coadd2d needs to be thought through. Currently turning it off.
         parset['flexure']['spec_method'] = 'skip'
+        # TODO This is currently the default for 2d coadds, but we need a way to toggle it on/off
+        parset['reduce']['findobj']['skip_skysub'] = True
```

Comment:
Do we need to warn users somehow that this is happening?  I.e., we don't want people putting something in their coadd parameter block an expect the code to respect it if we're hard-coding it this way.

Reply:
Well right now, co-adding non-skysubtracted frames is not supported, so I don't think it is a pressing issue.